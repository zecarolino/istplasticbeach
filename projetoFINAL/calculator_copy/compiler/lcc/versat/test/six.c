int main() {
	asm("\tldi 0x36\n\twrw CPRT_BASE\n");
	asm("\tldi 0xa\n\twrw CPRT_BASE\n");
	return 1;
}
