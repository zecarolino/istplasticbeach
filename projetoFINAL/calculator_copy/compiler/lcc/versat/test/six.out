VCD info: dumpfile xtop.vcd opened for output.
6
1
Catched Trap at time  375 clock cycles:
